### ISSUE TITLE
Is app-builder-bin being built with Go 1.14?

### ISSUE THREAD

**Issue by herbygillot** [develar/app-builder#37]
We're seeing the distributed version of app-builder-bin for macOS fail with an error that existed in Go 1.14: *SIGILL* on the `VZEROUPPER` instruction, as was discussed here: https://github.com/golang/go/issues/37459#issuecomment-591180408

Basically any binary built by that version of Go would fail when run on older amd64 processors on the `VZEROUPPER` instruction with a `SIGILL` (illegal instruction) panic.

We encountered this here: https://github.com/macports/macports-ports/pull/7195#issuecomment-633232936

Pertinent log files:
- https://build.macports.org/builders/ports-10.14_x86_64-builder/builds/60487/steps/install-port/logs/stdio
- https://build.macports.org/builders/ports-10.15_x86_64-builder/builds/31248/steps/install-port/logs/stdio

So basically, I'd like to know if the app-builder-bin binary being distributed in npm is being built with Go *1.14* instead of the very latest version, *1.14.3*.

**Comment by herbygillot** [comment]
This currently looks resolved.
